ocs_ci/ocs/resources/objectbucket.py

```diff
@@ -392,9 +396,79 @@ def __init__(self, *args, **kwargs):
         create_resource(**obc_data)
 
 
+class MCGNamespaceBucket(ObjectBucket):
+    """
+    Implementation of an MCG bucket using the S3 API
+    """
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.read_ns_resources = kwargs['read_ns_resources'] if 'read_ns_resources' in kwargs else ''
+        self.write_ns_resource = kwargs['write_ns_resource'] if 'write_ns_resource' in kwargs else ''
```

Comment:
I think it'll be simpler to replace it with a `.get()` - 
```suggestion
        self.write_ns_resource = kwargs.get('write_ns_resource')
```
Same regarding line 406

Reply:
fixed